src/live-common-set-supported-currencies.js

```diff
@@ -7,6 +7,7 @@ setPlatformVersion("0.0.1");
 setSupportedCurrencies([
   "bitcoin",
   "ethereum",
+  "elrond",
```

Comment:
As discussed can you please remove this for now?

This will allow to merge silently and the integration can still be tested using the env var: `EXPERIMENTAL_CURRENCIES=elrond`